config.yaml

```diff
@@ -1,14 +1,41 @@
 options:
-  string-option:
+  github-client:
```

Comment:
They were the initial targets. I think for now the idea is to integrate with the low hanging fruit and progress towards adding in the other integration paths like adding SAML/LDAP